testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -21,47 +21,48 @@
 #
 from __future__ import  absolute_import
 import MDAnalysis
-from MDAnalysisTests import module_not_found
+import pytest
 
-from numpy.testing import TestCase, assert_equal, assert_almost_equal, dec
+from numpy.testing import assert_equal, assert_almost_equal
 import numpy as np
 
 from MDAnalysisTests.datafiles import Martini_membrane_gro
 
-class TestLeafletFinder(TestCase):
-    def setUp(self):
-        self.universe = MDAnalysis.Universe(Martini_membrane_gro, Martini_membrane_gro)
-        self.lipid_heads = self.universe.select_atoms("name PO4")
-        self.lipid_head_string = "name PO4"
 
-    def tearDown(self):
-        del self.universe
-        del self.lipid_heads
-        del self.lipid_head_string
+class TestLeafletFinder(object):
+    @staticmethod
+    @pytest.fixture()
+    def universe():
+        return MDAnalysis.Universe(Martini_membrane_gro, Martini_membrane_gro)
 
-    def test_leaflet_finder(self):
+    @pytest.fixture()
+    def lipid_heads(self, universe):
+        return universe.select_atoms("name PO4")
+
+    lipid_head_string = "name PO4"
```

Comment:
This should be defined at the beginning, and the variable name should be in capital if the class get removed. It should certainly not be defined in the class scope, in between two methods.